applications/audio/broadcast_configuration_tool/src/main.c

```diff
@@ -0,0 +1,504 @@
+
+static struct bt_bap_lc3_preset lc3_preset_48_4_1 = BT_BAP_LC3_BROADCAST_PRESET_48_4_1(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+static struct bt_bap_lc3_preset lc3_preset_48_4_2 = BT_BAP_LC3_BROADCAST_PRESET_48_4_2(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+static struct bt_bap_lc3_preset lc3_preset_48_6_1 = BT_BAP_LC3_BROADCAST_PRESET_48_6_1(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+static struct bt_bap_lc3_preset lc3_preset_48_6_2 = BT_BAP_LC3_BROADCAST_PRESET_48_6_2(
+	BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT, BT_AUDIO_CONTEXT_TYPE_MEDIA);
+
+struct bap_preset {
+	struct bt_bap_lc3_preset *preset;
+	char name[PRESET_NAME_MAX];
+};
+
+static struct bap_preset bap_presets[] = {
+	{.preset = &lc3_preset_8_2_1, .name = "8_2_1"},
+	{.preset = &lc3_preset_8_2_2, .name = "8_2_2"},
+	{.preset = &lc3_preset_16_2_1, .name = "16_2_1"},
+	{.preset = &lc3_preset_16_2_2, .name = "16_2_2"},
+	{.preset = &lc3_preset_24_2_1, .name = "24_2_1"},
+	{.preset = &lc3_preset_24_2_2, .name = "24_2_2"},
+	{.preset = &lc3_preset_32_2_1, .name = "32_2_1"},
+	{.preset = &lc3_preset_32_2_2, .name = "32_2_2"},
+	{.preset = &lc3_preset_48_2_1, .name = "48_2_1"},
+	{.preset = &lc3_preset_48_2_2, .name = "48_2_2"},
+	{.preset = &lc3_preset_48_4_1, .name = "48_4_1"},
+	{.preset = &lc3_preset_48_4_2, .name = "48_4_2"},
+	{.preset = &lc3_preset_48_6_1, .name = "48_6_1"},
+	{.preset = &lc3_preset_48_6_2, .name = "48_6_2"},
+};
+
+/**
+ * @brief	Broadcast source static extended advertising data.
+ */
+static struct broadcast_source_ext_adv_data ext_adv_data[] = {
+	{.uuid_buf = &uuid_data,
+	 .pba_metadata_vacant_cnt = BROADCAST_SOURCE_PBA_METADATA_VACANT,
+	 .pba_buf = pba_data[0]},
+	{.uuid_buf = &uuid_data2,
+	 .pba_metadata_vacant_cnt = BROADCAST_SOURCE_PBA_METADATA_VACANT,
+	 .pba_buf = pba_data[1]}};
+#else
+/**
+ * @brief	Broadcast source static extended advertising data.
+ */
+static struct broadcast_source_ext_adv_data ext_adv_data[] = {{.uuid_buf = &uuid_data},
+							      {.uuid_buf = &uuid_data2}};
+#endif /* (CONFIG_AURACAST) */
+
+/**
+ * @brief	Broadcast source static periodic advertising data.
+ */
+static struct broadcast_source_per_adv_data per_adv_data[] = {{.base_buf = &base_data},
+							      {.base_buf = &base_data2}};
+
+/**
+ * @brief	Zbus listener to receive events from bt_mgmt.
+ *
+ * @param[in]	chan	Zbus channel.
+ *
+ * @note	Will in most cases be called from BT_RX context,
+ *		so there should not be too much processing done here.
+ */
+static void bt_mgmt_evt_handler(const struct zbus_channel *chan)
+{
+	int ret;
+	const struct bt_mgmt_msg *msg;
+
+	msg = zbus_chan_const_msg(chan);
+
+	switch (msg->event) {
+	case BT_MGMT_EXT_ADV_WITH_PA_READY:
+		LOG_INF("Ext adv ready");
```

Comment:
Would be great for EXT-ADV and PER-ADV ready.

Reply:
Not sure what are you are asking for here

Reply:
Since the event is BT_MGMT_EXT_ADV_WITH_PA_READY, so it is not only "EXT ADV ready".

Reply:
Ok, I see what you mean. The per adv is sort of in the ext adv, so in the context of this app there will never be an ext_adv without a per_avd, nor will there ever be a per_adv ready alone. We could probably also just remove this print all together